app/i18n/locales/pt-BR.json

```diff
@@ -763,5 +763,7 @@
   "Pinned_a_message": "Fixou uma mensagem:",
   "You_dont_have_permission_to_perform_this_action": "Você não tem permissão para realizar esta ação. Verifique com um administrador do espaço de trabalho.",
   "Jump_to_message": "Ir para mensagem",
-  "Missed_call": "Chamada perdida"
+  "Missed_call": "Chamada perdida",
+  "In_app_message_notifications": "Notificações de mensagens no aplicativo",
```

Comment:
This is a literal translation, but not accurate. AFAIK there's no pt-BR term for `in-app`. Same as push notification.
I disagree with the translation you're proposing because it creates confusion.
Which notification wouldn't be `no aplicativo`?
`Push` still belongs to the app.

IMO `Notificações de mensagens in-app` is better, although it's an anglicism.